src/main/java/ome/services/fulltext/FullTextBridge.java

```diff
@@ -245,6 +248,41 @@ public void set_annotations(final String name, final IObject object,
         }
     }
 
+    /**
+     * Walks the acquisition related metadata including channel names.
+     *
+     * @param name
+     * @param object
+     * @param document
+     * @param opts
+     */
+    public void set_acquisition(final String name, final IObject object,
+                                final Document document, final LuceneOptions opts) {
+        if (object instanceof Image) {
+            final Image image = (Image) object;
+            if (image.sizeOfPixels() == 0) {
+                return;
+            }
+            final Pixels pixels = image.getPrimaryPixels();
+            if (pixels == null) {
+                return;
+            }
+            final Iterator<Channel> channelIterator = pixels.iterateChannels();
+            while (channelIterator.hasNext()) {
+                final Channel channel = channelIterator.next();
+                final LogicalChannel logical = channel.getLogicalChannel();
+                if (logical != null) {
+                    addIfNotNull(document, "channel.name", logical.getName(), opts);
+                    addIfNotNull(document, "channel.fluor", logical.getFluor(), opts);
+                    addEnumIfNotNull(document, "channel.mode", logical.getMode(), opts);
```

Comment:
the emission wavelength is used for "label" i.e. value displayed in the UI
```
public String getChannelLabeling()
    {
        String value = getName();
        if (StringUtils.isNotBlank(value)) return value;
        value = getFluor();
        if (StringUtils.isNotBlank(value)) return value;
        Length v = null;
        try {
            v = getEmissionWavelength(null);
        } catch (BigResult e) { }
        if (v != null) {
        	return ""+ DoubleMath.roundToInt(v.getValue(), RoundingMode.DOWN);
        }
        return ""+index;
    }
```
So that value should probably be indexed

Reply:
As "channel.label"?

Reply:
label encapsulating the logic described above or just wavelength?

Reply:
can do either or both. So separating the questions:
 * do we want to introduce a user-facing field "channel.label" which is the `OR` of several other fields?
 * do we want to introduce `channel.wavelength` and if so, in what unit/format?